agriculture_shop/data/product_product.xml

```diff
@@ -5,15 +5,18 @@
         <field name="categ_id" ref="product_category_4"/>
         <field name="is_storable">True</field>
         <field name="list_price">12.0</field>
+        <field name="compare_list_price">14.0</field>
         <field name="image_1920" type="base64" file="agriculture_shop/static/src/binary/product_template/10-image_1920"/>
         <field name="description_sale">Fast Root Fertilizer is a potent and specialized formula designed to promote rapid root development in plants. Packed with essential nutrients and growth stimulants, this fertilizer accelerates the establishment of strong and healthy root systems, enhancing overall plant growth and vitality.</field>
         <field name="use_expiration_date" eval="True"/>
         <field name="expiration_time">10</field>
         <field name="available_in_pos" eval="True"/>
         <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_4')])]"/>
+        <field name="qty_available">20.0</field>
```

Comment:
Didn't know we could set the quantity available that way. But since it creates stock.quant, this should be done in demo/